EnvironmentSimulator/Modules/RoadManager/RoadManager.hpp

```diff
@@ -18,6 +18,7 @@
 #include <map>
 #include <vector>
 #include <list>
+#include <experimental/optional>
```

Comment:
That is for a bool variable that was missing in the classes that was needed for the opendrive specification. Since it's a bool it's not possible to check if it contains a valid value so I needed an optional to check whether it was initialized or not.

Since C++ 17 it is moved to <optional> but I didn't want to force the library to use that.